`interbit keys` command throwing an error

The `interbit keys` command throws the following error:

```
module.js:540
    throw err;
    ^

Error: Cannot find module 'watch'
    at Function.Module._resolveFilename (module.js:538:15)
    at Function.Module._load (module.js:468:25)
    at Module.require (module.js:587:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/interbit-cli/node_modules/interbit/src/file/watchCovenants.js:1:77)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
```

--- a/packages/interbit/package.json
+++ b/packages/interbit/package.json
@@ -17,7 +17,8 @@
     "json-stable-stringify": "^1.0.1",
     "lodash": "^4.17.5",
     "object-hash": "^1.3.0",
-    "path": "^0.12.7"
+    "path": "^0.12.7",
+    "watch": "^1.0.2"
   },
   "devDependencies": {
     "assert": "^1.4.1",

